students/mariusz_michalczyk/lesson_08_regex/strong_password_detection.py

```diff
@@ -0,0 +1,15 @@
+import re
+
+
+def check_password(password):
+    rgx_strip = re.compile(r'[0-9]+[a-zA-Z]{8,}|[a-zA-Z]{8,}[0-9]+')
```

Comment:
`abcd3aBc` should pass and won't. As they suggested, the easiest way is to use multiple validatiors. Say:
```
len_ok = len(pass) >= 8
digits_ok = re.search(r'\d', pass)
case_ok = ...
conditions = len_ok, digits_ok, case_ok
if all(conditions):
...
```